notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -479,8 +489,8 @@
         "id": "FJW4q-0jO2Xf"
       },
       "source": [
-        "### Create Features\n",
-        "Features can be created within each entity type. Add defining features to the `Users` entity type and `Movies` entity type by using the following methods."
+        "### Create Vertex AI Feature resources for the Vertex AI Entity Type resources\n",
+        "Vertex AI Feature resources can be created within each Vertex AI Entity Type. Add defining features to the `Users` entity type and `Movies` entity type by using the following methods."
```

Comment:
"You can create features within each entity type. Use the following methods to add defining features to the `users` and `movies` entity types."